### PR TITLE
Apim 2635 - release notes apim pipeline

### PR DESCRIPTION
**Pull request by jourdiw**
## Issue

https://gravitee.atlassian.net/browse/APIM-2635

## Description

Add release notes apim pipeline

### CODE REVIEW

File: .circleci/ci/src/pipelines/pipeline-release-notes-apim.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { CircleCIEnvironment } from './circleci-environment';
+import { Config } from '@circleci/circleci-config-sdk';
+import { ReleaseNotesApimWorkflow } from '../workflows';
+
+export function generateReleaseNotesApimConfig(environment: CircleCIEnvironment): Config {
+  const dynamicConfig = new Config();
```
Comment (phiz71) [inline .circleci/ci/src/pipelines/pipeline-release-notes-apim.ts:9]: I think we could add the check on graviteeioVersion

File: .circleci/ci/src/jobs/job-release-notes-apim.ts [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+        command: `npm run zx -- --quiet ci-steps/generate-changelog.mjs --version=${environment.graviteeioVersion}`,
+        working_directory: './release',
+      }),
+    ];
+
+    if (!environment.isDryRun) {
+      steps.push(
+        new commands.Run({
+          name: 'Get RELEASE_NOTES_PR_URL',
+          command: 'echo "export RELEASE_NOTES_PR_URL=$(cat /tmp/releaseNotesPrUrl.txt)" >> $BASH_ENV',
```
Comment (Okhelifi) [inline .circleci/ci/src/jobs/job-release-notes-apim.ts:63]: can we avoid the usage of BASH_ENV ?

File: .circleci/ci/src/jobs/job-release-notes-apim.ts [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+import { github } from '../orbs/github';
+import { Command } from '@circleci/circleci-config-sdk/dist/src/lib/Components/Commands/exports/Command';
+import { NodeLtsExecutor } from '../executors';
+import { config } from '../config';
+
+export class ReleaseNotesApimJob {
+  private static jobName: string = 'job-release-notes-apim';
+  public static create(dynamicConfig: Config, environment: CircleCIEnvironment): Job {
+    dynamicConfig.importOrb(keeper);
+    dynamicConfig.importOrb(slack);
```
Comment (phiz71) [inline .circleci/ci/src/jobs/job-release-notes-apim.ts]: We could import slack orb only if needed (i.e. !dryRun)

File: .circleci/ci/src/jobs/job-release-notes-apim.ts (hunk 2 above)
Comment (jourdiw) [reply .circleci/ci/src/jobs/job-release-notes-apim.ts:63]: I'm not sure because we recuperate the value from the `generate-changelog.mjs` actions

**Review by Okhelifi — APPROVED** [review]

File: .circleci/ci/src/jobs/job-release-notes-apim.ts (hunk 2 above)
Comment (jourdiw) [reply .circleci/ci/src/jobs/job-release-notes-apim.ts:63]: As discussed in Slack, we will leave it how it is right now and try to refactor this part later.

**Review by phiz71 — APPROVED** [review]

**Pull request merged**